### ISSUE TITLE
conv5 后做的是3x3窗口，这一步是卷积？

### ISSUE THREAD

**Issue by msxx001** [eragonruan/text-detection-ctpn#285]
你好，新版本的model_train.py model 中，VGG16以后 rpn_conv = slim.conv2d(conv5_3, 512, 3)，你这里做的是卷积，而原文的滑动窗口并不是做的卷积，是这样吗？

**Comment by centosrhel** [comment]
> 你好，新版本的model_train.py model 中，VGG16以后 rpn_conv = slim.conv2d(conv5_3, 512, 3)，你这里做的是卷积，而原文的3x3滑动窗口是为了利用中心点周边信息，产生3x3xC的向量，并不是做的卷积，是这样吗？

I agree with you. The implement here is the same as that in faster-rnn, so is more reasonable, i think.

**Comment by Hanko666** [comment]
所以这部分我们在跑的时候是不是需要修改一下子？还是说结果不会差很多？

**Comment by zparkgao** [comment]
这部分代码作者能解释下么？这个地方跟论文出处有点大

**Comment by LinBaoxin** [comment]
这边似乎都没有RPN啊，BiLSTM出来的结果直接送去算Loss了。而Caffe的deploy.prototxt里BiLSTM出来的结果送进了RPN去算offset和score。有大神能不吝赐教嘛？

**Comment by khinggan** [comment]
我也有这个疑惑，能解释一下吗？
